.github/workflows/on-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: Post-merge chores
+
+on:
+    push:
+        branches: 
+            - main
+
+jobs:
+    autoupdate:
+        name: Update outstanding PRs
```

Comment:
again, match the name and the id (in this case it's `autoupdate`, but maybe `update-pull-requests` would be better?